src/app/header/state/user/user.state.ts

```diff
@@ -0,0 +1,15 @@
+export interface UserState {
+  user: string,
+  isCreated: boolean,
+  showModal: boolean
```

Comment:
i would call this isModalVisible or isModalShown ... just to follow the same naming convention for booleans

Reply:
sure